packages/ffe-accordion-react/src/AccordionItem.js

```diff
@@ -36,7 +36,7 @@ class AccordionItem extends Component {
     toggle = () => {
         const open = this.getOpen();
         this.props.onToggleOpen(!open);
-        this.setState({ open: !open });
+        this.setState(prevState => ({ open: !prevState.open }));
```

Comment:
Is there a chance that `open` returned by `this.getOpen()` can get out of sync with `prevState.open` ?

For example if `toggle` is called many times when `setState`will queue up the changes?

Reply:
Good catch.